spid_cie_oidc/provider/settings.py

```diff
@@ -0,0 +1,47 @@
+
+from spid_cie_oidc.entity.utils import exp_from_now, iat_now
+
+JWT_CLIENT_ASSERTION = {
```

Comment:
Please, this is a test not an app settings

Am I wrong with that?
that's something for test purpose and not for general app configuration